### ISSUE TITLE
Problem about aux_loss

### ISSUE THREAD

**Issue by necrophagists** [OpenSparseLLMs/MoM#2]
The code seems like not add aux_loss to cross-entropy loss. Why?

**Comment by JusenD** [comment]
Hi, thanks for your attention. I separate the cross-entropy loss and aux loss to compare with other models' cross-entropy loss during training. To ensure proper integration of the auxiliary loss in backpropagation, I've made necessary adjustments to the transformers.trainer implementation by summing the losses before backward propagation. I will update the code and thank you!
